src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2145,38 +2160,30 @@ private void AddTrivia(CSharpSyntaxNode trivia, [NotNull] ref SyntaxListBuilder?
             list.Add(trivia);
         }
 
-        private bool ScanMultiLineComment(out bool isTerminated)
+        private void ScanMultiLineComment(out bool isTerminated, char delimiter)
         {
-            if (TextWindow.PeekChar() == '/' && TextWindow.PeekChar(1) == '*')
```

Comment:
Note: all callers have already verified this. I've turned the condition into a `Debug.Assert` instead.